apilint/build.gradle

```diff
@@ -17,20 +22,32 @@ sourceSets {
     }
 }
 
-group 'org.mozilla.android'
+group Config.GROUP
 version '0.1'
 
-buildDir "${topobjdir}/apilint"
-
 gradlePlugin {
     plugins {
         apilint {
-            id = 'org.mozilla.apilint'
+            id = Config.GROUP
             implementationClass = 'org.mozilla.apilint.ApiLintPlugin'
         }
     }
 }
 
+pluginBundle {
+    website = 'https://github.com/mozilla-mobile/gradle-apilint'
+    vcsUrl = 'https://github.com/mozilla-mobile/gradle-apilint'
+
+    tags = ['api', 'lint', 'mozilla', 'plugin', 'compatibility']
+    description = 'Tracks the API of an android library and helps maintaining backwards compatibility.'
```

Comment:
nit: Android (capital "A").

Reply:
And maybe "helps maintain backward compatibility."

Reply:
thanks for fixing my spelling :) fixed.